.github/workflows/publish-op-node.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow will publish a github release for op-node
+
+name: Publish
+run-name: ${{ github.actor }} is publishing an op-node release 🚀
+on:
+  push:
+    tags:
+      - 'op-node/v*'
+
+# Always wait for previous release to finish before releasing again
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        arch: [amd64, arm64]
+        exclude:
+          - os: ubuntu-latest
+            arch: arm64
+    env:
+      BUILD_DIR: ${{ github.ref_name }}
+      BIN_DIR: ${{ github.ref_name }}/build/bin
+    steps:
+      - name: Set file name
+        run: |
+          # replace "op-node/v*" with "op-node.v*" since `actions/upload-artifact@v4` doesn't support slashes in file names
+          FILE_NAME="op-node.${GITHUB_REF_NAME:8}.${{ matrix.os == 'ubuntu-latest' && 'linux' || 'darwin' }}-${{ matrix.arch }}.tar.gz"
```

Comment:
Quick question: what does `:8` mean in `${GITHUB_REF_NAME:8}`, slicing?

Reply:
It returns the substring starting from index 8, e.g., for `op-node/v1.0`, it returns `v1.0`.